libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.cpp

```diff
@@ -252,14 +252,31 @@ void AC_AttitudeControl_Multi::update_throttle_rpy_mix()
 
 void AC_AttitudeControl_Multi::rate_controller_run()
 {
+	rate_controller_run(false,0);
+}
+
+void AC_AttitudeControl_Multi::rate_controller_run(bool rotational_frame, float instant_heading)
+{
+	Vector3f gyro_latest = _ahrs.get_gyro_latest();
```

Comment:
const all this and fix indentation

Reply:
ok for the const, but what is the problem with indentation?

Reply:
mix between space and tabs, so it looks like it isn't well indented